src/augments/number.ts

```diff
@@ -128,7 +128,7 @@ export function snapToLowerMultipleOfPowerOfTen(input: number): number {
         negativeModifier = -1;
         input = input * -1;
     }
-    const stringNumber = String(input);
+    const stringNumber = String(input.toFixed(0));
```

Comment:
```suggestion
    const stringNumber = input.toFixed(0);
```
`String()` no longer necessary.